packages/react-composites/src/composites/MeetingComposite/NotificationIcon.tsx

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import React from 'react';
+import { Icon, Stack } from '@fluentui/react';
+import { NotificationIconStyles, NotificationTextStyles } from './styles/MeetingCompositeStyles';
+
+/**
+ * @private
+ */
+export type NotificationIconProps = {
+  numberOfMessages: number;
```

Comment:
can we be consistent here as well.

Messages or ChatMessages?

should we use prefix with number or a suffix with count?